File: android-app/transfer-api/src/main/java/com/hlrms/transferapi/dto/ExchangeRatesResponse.java
package com.hlrms.transferapi.dto;

import java.math.BigDecimal;
import java.time.Instant;
import java.util.Map;

public record ExchangeRatesResponse(

        Instant updatedAt,

        Instant nextUpdateAt,

        String provider,

        Map<String, Map<String, BigDecimal>> rates

) {
}

File: android-app/app/src/main/java/com/hlrms/mobile/data/remote/exchange/ExchangeRatesResponse.kt
package com.hlrms.mobile.data.remote.exchange

import java.math.BigDecimal

data class ExchangeRatesResponse(

    val updatedAt: String,

    val nextUpdateAt: String,

    val provider: String,

    val rates:
        Map<String, Map<String, BigDecimal>>
)
